src/Workspaces/CSharp/Portable/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -69,6 +69,7 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
                             !currentToken.IsParenInArgumentList() &&
                             !currentToken.IsDotInMemberAccess() &&
                             !currentToken.IsCloseParenInStatement() &&
+                            !currentToken.IsCloseParenInTuple() &&
```

Comment:
💡 Consider moving this up next to the condition added in 4e62713115d32cc5a5cdf01a9586f43207c4c76e, considering how related this is. Also, I would consider that commit a model for correctness for testing strategy and implementation patterns.

Reply:
Good suggestion, I've moved it to beneath the `IsCommaInTupleExpression()` invocation.

I'm a little bit confused about what you mean with the latter sentence. Do you mean that it's becoming a bit too specific, and that the other case can be used as guidance for generalizing this?, _In case_, I'd want that.